src/components/Calculator.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './Calculator.css';
+
+const Calculator = () => (
+  <div className="calculator-grid">
+    <div className="calculator-output">123</div>
+    <button type="button">AC</button>
+    <button type="button">+/-</button>
+    <button type="button">%</button>
+    <button type="button" className="yellow-btns">/</button>
```

Comment:
- [ ] This is important for future projects. Kindly use the division symbol in the design, which is &div;. You can use entity code for division if you do not have that sign on your keyboard.